engine_details/development/debugging/profiling/tracing_profilers.rst

```diff
@@ -0,0 +1,221 @@
+.. _doc_tracing_profilers:
+
+Tracing Profilers
+=================
+
+Godot currently supports two tracing profilers: Tracy and Perfetto.
+
+In order to use either of them, you'll need to build the engine from source.
+If you've never done this before, please read
+:ref:`these docs <doc_compiling_index>` for the platform you want to profile on.
+You'll need to perform the same steps here, but with the addition of a single
+extra argument for ``scons``.
+
+.. _doc_tracy_profiler:
+
+Tracy for Windows, Linux and MacOS
+----------------------------------
+
+Tracy is an Open Source profiler that runs on a wide variety of platforms,
+including Windows, Linux and MacOS. While it is primarily a tracing profiler,
+it can also periodically sample data like a
+:ref:`sampling profiler <doc_sampling_profilers>`, giving some of the benefits
+of both approaches.
+
+Build Godot with Tracy support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, clone the latest version of the Tracy source code ("0.13.0" at the
+time of this writing) using Git:
+
+.. code-block:: shell
+
+    git clone -b v0.13.0 --single-branch https://github.com/wolfpld/tracy.git
+
+This will create a ``tracy`` directory - you can place this anywhere.
+
+Next, build the release templates for your platform using ``scons``, but adding
+the ``profiler_path=path/to/tracy`` argument with the real path to the ``tracy``
+directory, as well as ``debug_symbols=yes`` to allow Tracy's sampling features
+to work.
+
+.. note::
+
+    You don't have to build release templates, you could also build debug
+    templates, or even the editor. However, it's generally recommended to
+    profile release templates, because that is the version your players will
+    use, and it will perform differently than other types of builds.
+
+For example, to build release templates for Windows:
+
+.. code-block:: shell
+
+    scons platform=windows target=template_release debug_symbols=yes profiler_path=path/to/tracy
+
+Get the Tracy "server"
+~~~~~~~~~~~~~~~~~~~~~~
+
+In Tracy terminology, the application you are profiling is the "client", and
+the one receiving the data is the "server".
+
+If you are on Windows, you can download a pre-built ``tracy-profiler.exe``
+from the Tracy `releases page <https://github.com/wolfpld/tracy/releases>`_.
+
+However, if you're on Linux or MacOS, you'll need to build it yourself.
```

Comment:
I actually had trouble building tracy at first, but later found out both `brew` and `nix` supply pre-built tracy binaries. I would not recommend building it yourself by default.

```suggestion
If you're on Linux or macOS, you may be able to get a Tracy binary from a package manager.
For example, on macOS, you can install Tracy using homebrew:

.. code-block:: shell

    brew install tracy

Alternatively, you can build the Tracy server yourself, as explained below.
```

Reply:
It won't work if installed `tracy` version is not matching the client (or at least protocol version). Current `brew` version (0.11.1) doesn't work with current GitHub release client (0.13).

Reply:
Thanks!

In my latest update, I've mentioned that you might be able to find a pre-built binary from a package manager, but that if you do, you should be careful to match the version